src/public/modules/users/controllers/authentication.client.controller.js

```diff
@@ -195,21 +193,22 @@ function AuthenticationController(
           vm.signInMsg.isMsg = false
           vm.showOtpDelayNotification = true
         }, 20000)
-      },
-      function (error) {
+      })
+      .catch((error) => {
+        const errorMsg = get(
+          error,
+          'response.data',
```

Comment:
this endpoint returns `{ message: string }` if there's an error right, so this should be `'response.data.message'`?

Reply:
Good catch, introduced when catching verify email domain's response... we should have fixed that in v3 API call T_T

Fixed in f429926